src/Eurofurence.App.Server.Services/Events/EventService.cs

```diff
@@ -28,5 +57,232 @@ public IQueryable<EventRecord> FindConflicts(DateTime conflictStartTime, DateTim
                 e.StartDateTimeUtc <= queryConflictEndTime &&
                 e.EndDateTimeUtc >= queryConflictStartTime);
         }
+
+        public async Task RunImportAsync()
+        {
+            _logger.LogInformation(LogEvents.Import, "Starting events import.");
+
+            var httpClient = new HttpClient();
+            var fileStream = await httpClient.GetStreamAsync(_configuration.Url);
+            TextReader reader = new StreamReader(fileStream);
+
+            using var csv = new CsvReader(reader, new CsvConfiguration(CultureInfo.CurrentCulture) { Delimiter = "," });
```

Comment:
I don't know if `CultureInfo.CurrentCulture` is a good idea here, since it means, use the computer's language to parse stuff and I think that it affects parsing the `TimeSpan`s in `EventImportRow`. Using `CultureInfo.InvariantCulture` should make it use US-like style which is good enough for just the `TimeSpan`s.